boards/arm/nrf9131ek_nrf9131/Kconfig.defconfig

```diff
@@ -0,0 +1,53 @@
+# nRF9131 EK NRF9131 board configuration
+
+# Copyright (c) 2023 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_NRF9131EK_NRF9131 || BOARD_NRF9131EK_NRF9131_NS
+
+config BOARD
+	default "nrf9131ek_nrf9131"
+
+# By default, if we build for a Non-Secure version of the board,
+# enable building with TF-M as the Secure Execution Environment.
+config BUILD_WITH_TFM
+	default y if BOARD_NRF9131EK_NRF9131_NS
+
+if BUILD_WITH_TFM
+
+# By default, if we build with TF-M, instruct build system to
+# flash the combined TF-M (Secure) & Zephyr (Non Secure) image
+config TFM_FLASH_MERGED_BINARY
+	bool
+	default y
+
+endif # BUILD_WITH_TFM
+
```

Comment:
This should be removed after #63515 is merged.

Reply:
done